messages/lightning.preview.app.md

```diff
@@ -1,21 +1,19 @@
 # summary
 
-Preview Lightning Experience Applications.
+Preview a Lightning Experience application locally and in real-time, without deploying it.
 
 # description
 
-Preview Lightning Experience Applications in real-time.
+In developer preview mode, see local changes to your app in a real-time preview that you don't have to deploy or manually refresh. To let you quickly iterate on your Lightning web components (LWCs) and pages, your app preview automatically refreshes when source code changes are detected.
 
-In dev preview mode, you can edit local files and see these changes to your Lightning Web Components (LWC) within your {org name} org:
+When you edit local files in developer preview mode, your LWCs reflect these changes within your {org name} org:
 
 - Basic HTML and CSS edits
-- Importing new CSS-only LWC
+- Importing new CSS-only LWCs
 - JS edits in-service component library
-- JS method changes in the LWC component that don't alter its public API.
+- JS method changes that don't affect the LWC's public API
 
-Other local changes require deployment to your org. However, changes made directly in your org (like modifying component properties and saving) are immediately live and won't show in your local files until you retrieve them from the org.
-
-This feature enables developers to quickly iterate on their components and pages, seeing the impact of changes in real-time without needing to deploy or refresh manually. Live reload is enabled by default to automatically refresh the preview when source code changes are detected.
+To see any other local changes in your app, you have to deploy to your org. If you make changes directly in your org, (like saving new component properties,) they're immediately live on your app. To update your local version of the app with those changes, you have to retrieve them from your org.
```

Comment:
Disclaimer: I may be overthinking this.

I feel like the two separate ideas conveyed in this paragraph need some separation, for clarity and distinction. My proposal is a bulleted approach that's prefaced with a "limitations" statement:

> There are currently a couple of limitations to consider:
> 1. To see any local changes in your app that are not listed above, you have to deploy to your org. [+1 for @jshackell-sfdc 's call-outs to the CLI commands for this, in the comment above.]
> 2. If you make changes directly in your org, (like saving new component properties), they're immediately live on your app. To update your local version of the app with those changes, you have to retrieve them from your org.

I'm concerned that, if these aren't called out more distinctly, both of the messages are going to get lost or muddled.

Reply:
@khawkins are the names "Lightning Preview" and "developer preview mode" interchangeable? Or is there one "official" name that I should use for consistency?

Reply:
@moorejacqueline It's a bit of a guess for me as to why the original copy said "developer preview mode". I think it may have been calling out that you have to have your org set to `DEVPREVIEW` mode in order to use this feature (among other qualifications), but that's not worth calling out in this description—which is why I was with Juliet on making it more meaningful.

Alas, I don't think there is as yet an official name for this. I believe it's in the works.